### ISSUE TITLE
`vcrash` breaks on machines with all-digits hostnames

### ISSUE THREAD

**Issue by AdamBromiley** [netkit-jh/netkit-jh-build#71]
**Describe the bug**
Running `vcrash` on a machine with an all-digit hostname (such as `1`) makes the `findVM()` function treat the number as a machine's PID, not its hostname. This leads to a failure in crashing the machine.

```bash
findVM() {
   ...
   if echo $1 | grep -qE "^[0-9]+$"; then
      # This is a virtual machine PID
```

**To Reproduce**
1. `vstart 1`
2. `vcrash 1`
3. See error

**Expected behavior**
Netkit should either prevent the creation of all-digit machine names on startup, or fix the `findVM` function.

**Comment by AdamBromiley** [comment]
Extending this issue to general lack of checking machine-name validity:
```
adam@HELIUM://opsecfailed/netkit-testlab$ ls
--help  --help.startup  lab.conf  lab.dep
opsecfailed@HELIUM://opsecfailed/netkit-testlab$ lstart

======================== Starting lab ===========================
Lab directory: /home/opsecfailed/netkit-testlab
Version(s):       <unknown>
Author(s):        <unknown>
Email(s):         <unknown>
Web(s):           <unknown>
Description(s):
<unknown>

Having issues starting the lab? Run 'lclean' before running 'lstart'.
=================================================================
You chose to use parallel startup.
Usage: make [options] [target] ...
Options:
  -b, -m                      Ignored for compatibility.
  -B, --always-make           Unconditionally make all targets.
  -C DIRECTORY, --directory=DIRECTORY
                              Change to DIRECTORY before doing anything.
  -d                          Print lots of debugging information.
```

**Comment by AdamBromiley** [comment]
Issue has been resolved in version 2.0.0-beta.
